src/lib/Toast/properties.ts

```diff
@@ -0,0 +1,23 @@
+export type ToastType = 'success' | 'error' | 'info' | 'warn';
+
+export type ToastProperties = {
+  type: ToastType;
+  message: string;
+  backgroundColor?: string;
+  textColor?: string;
+  leftIcon?: string | null;
```

Comment:
Lets use `?` or `null`. One of them are sufficient to indicate optional params

Reply:
updated.